<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>575</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>계정별검색보기</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>3</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>기타소득 상세 보기</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색대상년도</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>30</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>117</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>합계금액 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="hap_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>114</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="result_tableWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>160</y>
     <width>701</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="serch_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QComboBox" name="year_combo_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>71</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_cost_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>98</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 수정 저장하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serch_Button_2">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>50</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>프린터</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>75</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>구 분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_combo_widget">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>73</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serch_Button</tabstop>
  <tabstop>result_tableWidget</tabstop>
  <tabstop>hap_widget</tabstop>
  <tabstop>reset_Button</tabstop>
  <tabstop>close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>serch_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>serch_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serch_Button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>other_income_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_cost_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>edit_other_income()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>serch_account()</slot>
  <slot>reset()</slot>
  <slot>edit_other_income()</slot>
  <slot>other_income_print()</slot>
 </slots>
</ui>
